app/controllers/faculty_access_controller.rb

```diff
@@ -1,7 +1,7 @@
 class FacultyAccessController < ApplicationController
```

Comment:
@Dantemss not sure there is still a path to this controller - I think the faculty access application process is handled in `educator_signup_controller`